### ISSUE TITLE
Fail import w/cython, cffi and build on 3.5 HEAD.

### ISSUE THREAD

**Issue by Carreau** [zeromq/pyzmq#672]
Various tracebacks available [here](https://gist.github.com/Carreau/e433f5c972016287fb1c) depending on each case. 

Ok, it's Python 3.5 head, so I should partially blame myself.
And yes I'll try cython master.

**Comment by minrk** [comment]
Thanks, my guess is it's a Cython issue, but bundled libzmq is a C extension that doesn't involve Cython, so there may need to be changes there.

**Comment by Carreau** [comment]
Hum, indeed, I narrowed it down to : 
``` 
In [1]: import Cython.Compiler.Scanning
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-8faf04ac6b88> in <module>()
----> 1 import Cython.Compiler.Scanning

ImportError: dynamic module does not define module export function (PyInit_.Scanning)
```

**Comment by Carreau** [comment]
So it seem to be due to [this](https://hg.python.org/cpython/rev/e729b946cc03)  that finish to implement [pep 489](https://www.python.org/dev/peps/pep-0489/) more info on [issue tracker](http://bugs.python.org/issue24268)

**Comment by minrk** [comment]
This is a bug [in Python 3.5b1](https://bugs.python.org/issue24285) that will be fixed in 3.5b2. I tested with HEAD, and it works fine. There's nothing for us to do here.
